DigitalLearningSolutions.Web/Helpers/AccountDetailsDataHelper.cs

```diff
@@ -20,6 +20,10 @@ int centreId
                 formData.FirstName!,
                 formData.LastName!,
                 formData.Email!,
+                formData.HasProfessionalRegistrationNumber == true

```

Comment:
@SteveJacksonSoft note this additional change here.
This is to prevent saving a PRN when it is not required or invalid.
Example:
Click `yes` then add an invalid PRN.
Then click `no`.
Saving will set that invalid prn on the account.
This change prevents that, if the PRN question was answered with a `no`.

Reply:
Oho, good catch! And thanks for the clarification.